dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ManualRegistry.java

```diff
@@ -61,4 +63,9 @@ public Registry getRegistry() {
     public boolean canConvertToAuto(Tool tool) {
         return false;
     }
+
+    @Override
+    public Tool getToolFromNamespaceAndRepo(final String organization, final String repository) {
+        throw new CustomWebApplicationException("Unsupported operation, only Quay.io is supported for now", HttpStatus.SC_BAD_REQUEST);
```

Comment:
It would be nice if this string were a constant. That way if we add support for DockerHub but not manual, we only have to update the string in one location